src/css/archwizard.scss

```diff
@@ -1,9 +1,97 @@
-// color definitions
-$wz-color-default: #E6E6E6 !default;
-$wz-color-current: #808080 !default;
-$wz-color-done: #339933 !default;
-$wz-color-optional: #38ef38 !default;
-$wz-color-editing: #FF0000 !default;
+// Wizard layouts to be generated (all by default).
+// Override this variable to exclude layouts you don't need.
+$wz-layouts: 'horizontal' 'vertical' !default;
+
+// Wizard indicator styles to be generated (all by default).
+// Override this variable to exclude indicator styles you don't use.
+$wz-indicator-styles: 'small', 'large-filled', 'large-empty', 'large-filled-symbols', 'large-empty-symbols' !default;
+
+// Color definitions
+//
+// This is a mapping from indicator style and wizard step state to colors.
+//
+// Predefined indicator styles are: 'small', 'large-filled', 'large-empty', 'large-filled-symbols', 'large-empty-symbols'.
+//
+// Wizard step states are: 'default', 'current', 'done', 'optional', 'editing'.
+//
+// Colors are: 'border-color-default', 'background-color-default', 'symbol-color-default', 'border-color-hover',
+// 'background-color-hover', 'symbol-color-hover'.
+//
+// Notes:
+// - The order of wizard step states in this mapping does not affect the order of resulting CSS rules, which is fixed
+//   internally by angular-archwizard.
+// - Depending on the active indicator style, some colors may not be used.  For example, 'background-color-default' and
+//   'background-color-hover' do not make sense in 'large-empty-symbols' indicator style.
+// - You can set a color to `null` to exclude the corresponding property from CSS output.

```

Comment:
Where should I set the color to `null` to exclude the property from the CSS output?

Reply:
Added an example in f57cf94